templates/techreport/category.html

```diff
@@ -0,0 +1,117 @@
+{% extends "techreport/report.html" %}
+
+{% block filters %}
+<form id="page-filters">
+  <div>
+    <fieldset class="technology-filters">
+      <legend class="form-title">Choose a dataset</legend>
+
+      <div class="tech-selector">
+        <label for="categories_selector" class="tech">Category</label>
+        <div class="tech-input-wrapper">
+          {% set category_selected = tech_report_page.filters.category or tech_report_config.default_category %}
+          <select name="categories" class="tech" data-selected="{{ category_selected }}" id="categories_selector">
+            <option value="{{ category_selected }}" selected="true">
+              {{ category_selected }}
+            </option>
+          </select>
+        </div>
+      </div>
+
+    </fieldset>
+    <fieldset class="lens">
+      <legend class="sr-only">Lens</legend>
+
+      <label for="geo">Geo</label>
+      <div class="select-label">
+        <select name="geo" id="geo">
+          <option value="ALL" selected="true">
+            ALL
+          </option>
+        </select>
+      </div>
+
+      <label for="rank">Rank</label>
+      <div class="select-label">
+        <select name="rank" id="rank">
+          <option value="ALL" selected="true">
+            ALL
+          </option>
+        </select>
+      </div>
+    </fieldset>
+  </div>
+  <button type="submit" id="submit-form">Update</button>
+  {% include "techreport/components/filter_meta.html" %}
+  <fieldset class="breakdown">
+    <legend>Breakdown</legend>
+    <div class="client-breakdown-wrapper">
+      <label for="client-breakdown">Client</label>
+      <select name="client-breakdown" id="client-breakdown">
+        {% if request.args.get('client', '') == 'desktop' %}
+        <option value="mobile">Mobile</option>
+        <option value="desktop" selected>Desktop</option>
+        {% else %}
+        <option value="mobile" selected>Mobile</option>
+        <option value="desktop">Desktop</option>
+        {% endif %}
+      </select>
+    </div>
+  </fieldset>
+</form>
+
+<template id="filter-option">
+  <option value="Option">
+    Option
+  </option>
+</template>
+
+<template id="tech-selector">
+  {% set technology = "ALL" %}
+  {% set name = "tech-new" %}
+  {% set key = 2 %}
+  {% include "techreport/templates/selector_tech.html" %}
+</template>
+{% endblock %}
+
+{% block section %}
+  {{ super() }}
+  <div class="report-content">
+    <div class="info-panel">
+      <div class="block-l">
+        <p class="info-label">Beta</p>
+        <p>We're still actively working on the categories page.</p>
+      </div>
+    </div>
+
+    <div class="intro block-m">
+      <div class="info">
```

Comment:
When the API supports it, we should also have a placeholder for the category description here.